src/sgame/components/SpikerComponent.cpp

```diff
@@ -4,13 +4,14 @@
 
 static Log::Logger logger("sgame.spiker");
 
+static Cvar::Cvar<bool> sg_spiker_experiments( "sg_spiker_experiments", "wether to use experimental reloading for spikers or not", Cvar::NONE, true );
 /** Delay between shots. */
 constexpr int   COOLDOWN = 5000;
 /** Shoot prematurely whenever this much damage is expected. */
 constexpr float DAMAGE_THRESHOLD = 60.0f;
 /** Total number of missiles to shoot. Some might be skipped if they endanger allies.
  *  Actual values is +/- MISSILEROWS. */
-constexpr int   MISSILES = 26;
+static Cvar::Cvar<int> sg_spiker_missiles( "sg_spiker_missiles", "number of missiles spikers have", Cvar::NONE, 39 );
```

Comment:
If this was kept, then it should be `g_spikerMissiles` as opposed to `sg_spiker_missiles`.

I am wary of allowing server operators to fine tune **combat** balance values without packaging a mod. This runs the risk of giving everyone, including us developers, an unclear picture of the current state of the game. For players in particular it runs the risk of producing "false muscle memory" (not being able to intuitively judge your own and the enemies strength). This is also not done for any other setting that immediately affetcs damage, afair. So I think this should either remain hardcoded or be a field in a config file so that at least one can tell that they are playing on an arbitrarily modded server.

Reply:
> If this was kept, then it should be g_spikerMissiles as opposed to sg_spiker_missiles.

That's a nitpicking, but sure.

> I am wary of allowing server operators to fine tune combat balance values without packaging a mod. This runs the risk of giving everyone, including us developers, an unclear picture of the current state of the game. For players in particular it runs the risk of producing "false muscle memory" (not being able to intuitively judge your own and the enemies strength). This is also not done for any other setting that immediately affetcs damage, afair. So I think this should either remain hardcoded or be a field in a config file so that at least one can tell that they are playing on an arbitrarily modded server.

You are wrong here.
This code being on sgame means there is no need for a mod owner to share the change. Using a cvar for this is even the only way to communicate about the change in a reliable manner.
We also have seen the effect of that conservatism politics: no changes were done for years on 1st thrown values such as the ground flame stuff.
To balance things more easily, we need to enable server owners to change them without recompiling anything.

The "muscular memory" you mention is a decent argument, though, but I do know rather old games such as red-eclipse who indicate how much a server derives from the official settings. We could, and likely *should* do that, too, possibly in a better way, but this is out of topic.